### DISCUSSION TITLE
lil help

### DISCUSSION THREAD

**Discussion by zenfonseka** [radix-ui/website#274] [Q&A]
hey ,

i was very enjoyed your UI website and i loved it! but when i get it to Visual Code and run the command of "yarn dev" it's not running tho? can you help me to how to use it please? 

thank you!.

**Comment by peduarte** [comment]
Did you do just `yarn` first to install the dependencies?

**Comment by jjenzz** [comment]
@zenfonseka What version of `node` are you using? If you have `nvm`, try `nvm use 15.0.1` and see if that fixes things for you?
